move-to-infra/move-to-infra.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+# This script moves ARO apps into intrastructure nodes.
+# It follows RHEL guide: https://access.redhat.com/solutions/5034771
+#
+# Infra nodes are:
+#  label: node-role.kubernetes.io/infra: ""
+#  taints:
+#  - key: infra
+#    value: reserved
+#    effect: NoSchedule
+#  - key: infra
+#    value: reserved
+#    effect: NoExecute
+
+
+# Check if pods are running
+checkStatus()
+{
+  LABEL="$1"
+  NAMESPACE="$2"
+  EXPECTED_PODS="$3"
+  
+  echo "...checking pod labels ${LABEL} at namespace ${NAMESPACE}..."
+  ATTEMPTS=0
+  until [[ $ATTEMPTS -ge 10 ]]
```

Comment:
I'd recommend to 'wrap' variable names with curly brackets to prevent possible issues. E.g. ${ATTEMPTS}

Reply:
updated